server/controllers/Users.js

```diff
@@ -0,0 +1,52 @@
+import bcrypt from 'bcryptjs';
+import models from '../database/models';
+import {
+  serverResponse,
+  serverError,
+  checkEmail,
+  checkUserName
+} from '../helpers';
+
+const { User } = models;
+
+/**
+ * @export
+ * @class Users
+ */
+class Users {
+  /**
+   * @name signUp
+   * @async
+   * @static
+   * @memberof Users
+   * @param {Object} req express request object
+   * @param {Object} res express response object
+   * @returns {JSON} JSON object with details of new user
+   */
+  static async signUp(req, res) {
+    try {
+      const { password, confirmPassword } = req.body;
+      if (password !== confirmPassword) {
+        return serverResponse(res, 400, { error: 'Passwords do not match' });
+      }
+      if (await checkEmail(req.body.email)) {
+        return serverResponse(res, 409, { error: 'Email already in use' });
```

Comment:
change to 'Email has already been taken'

Reply:
Ok. I'll do that right away